clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -120,7 +135,12 @@ func (a *apiResponsivenessGatherer) Gather(executor QueryExecutor, startTime tim
 		return nil, err
 	}
 
-	badMetrics := a.validateAPICalls(config.Identifier, allowedSlowCalls, apiCalls)
+	ct, err := getCustomThresholds(config, apiCalls)
+	if err != nil {
+		return nil, err
+	}
+
+	badMetrics := a.validateAPICalls(config.Identifier, allowedSlowCalls, apiCalls, ct)
```

Comment:
Let's stick to existing convention and use full name, so s/ct/customThresholds/g

Reply:
Done. Thanks for the suggestion - I wasn't aware that we can name a variable with a name of a type in Golang ;).